CONTRIBUTING.md

```diff
@@ -8,3 +8,17 @@ Logging an [issue](https://github.com/superlou/fountain-atom/issues) is a welcom
 When submitting a pull request to address an existing issue, please be sure to run all of the spec tests from the project's root directory: `atom --test ./spec/*`
 
 Fountain files to support testing can be found in the `spec/test_files` directory.  New fountain test files may be added to this directory as needed, provided they fall under appropriate licensing.
+
+## Contribute to Fountain for Atom
+First of all, you have to fork the repository.  
+
+**Mac or Linux**  
+`cd ~/.atom` *Go to the atom folder*  
+`mkdir -p dev/packages` *Creates the folder tree for development*  
+`atom -d dev/packages/` *Go to the newly created folder*  
+`git clone https://github.com/YourPseudo/fountain-atom.git` *Clone your forked repository in the current folder*  
+`cd atom-fountain` *Go to the cloned folder*  
```

Comment:
There's actually an atom command that can replace the above steps.  I'm not in a position to look it up ATM.  But it basically creates the directory for you and creates a symbolic link to your forked local copy.  Probably safer than cloning directly into the dev packages dir.

Reply:
I looked around about that, and found the right steps to follow : https://flight-manual.atom.io/hacking-atom/sections/contributing-to-official-atom-packages/  
Then I modified my commit. I don't know if I did it correctly, I did `--amend` it and `push -f`.
It seems that the pull request is updated, but I'm still a noob in pull request procedure.